app/services/price_policies/time_based_price_calculator.rb

```diff
@@ -40,6 +45,47 @@ def cost_and_subsidy(duration_mins, discount_multiplier)
       end
     end
 
+    def duration_pricing_cost_and_subsidy(duration_mins)
+      duration_in_hours = duration_mins / 60.0
```

Comment:
I think we should probably be storing duration rate `rate` and `subsidy` per minute, for consistency with `usage_rate` and `usage_subsidy`

Reply:
So we should have the user set the hourly rate or subsidy and then modify it before storing? I think that's what we do for `usage_rate`

Reply:
Yeah, I think there are benefits to being consistent even though it adds some complexity up front